scripts/docker/build.sh

```diff
@@ -258,10 +270,26 @@ if [ "$astrobee_quick" = "true" ]; then
     build astrobee_quick "${revision}-" "quick-"
 fi
 
+if [ "$rolling_base" = "true" ]; then
+    build ros2/ros2_rolling_base "${revision}-" "rolling-base-"
```

Comment:
In order to be kind to anyone who hopes to parse these image names later, maybe it would be better to not allow an internal hyphen in the last arg to `build`. Like call it `rollingbase-` or `rolling_base-` instead of `rolling-base-`? Not critical, just a suggestion.